README.md

```diff
@@ -2,58 +2,6 @@
 
 This hosts multiple scripts necessary for filtering and processing of variant calls in the vcfs/txt file generated by callers.
 
-## Callers Supported
-`pv` is the main command for the `postprocessing_variant_calls` package see `pv --help` to see supported variant callers commands. 
-
-### VarDictJava
-
-The sub-command `pv vardict` allows users to perform post-processing on VarDictJava output. The two supported inputs to `pv vardict` from VarDictJava are `single` and `case-control` vcfs. 
-
-To specify to `pv vardict`, which input type will be used one of the following sub-commands may be used: 
-- `pv vardict single` for single sample vcfs 
-- `pv vardict case-control` for case-controlled vcfs. 
-
-Next the user can specify, what post-processing should be done. Right now, `postprocessing_variant_calls` supports filtering: 
--  `pv vardict single filter` 
--  `pv vardict case-control filter` 
-
-Finally, we can specify the paths and options for our filtering and run our command. Here is an example using the test data provided in this repository: 
-
-`pv vardict single filter --inputVcf data/Myeloid200-1.vcf  --tsampleName Myeloid200-1  -ad 1 -o data/single`
-
-There are various options and input specifications for filtering so see `pv vardict single filter --help` or `pv vardict single case-sontrol --help` for help. 
-
-See `example_calls.sh` for more example calls. 
-
-### Maf 
-
-maf concat examples: 
```

Comment:
again very minor, but maybe adding a sentence at the beginning of the README (similar to line 10) with a quick explanation to what the category of 'maf' commands are intended to do in post-processing. When the vcf2maf module has been approved in msk-modules, we could even add a hyperlink to it in the README here

Reply:
similar to above I added a link to the README that points to `docs/maf`